hit_approve_bonus.py

```diff
@@ -0,0 +1,82 @@
+from connect import *
+
+import psycopg2.extras
+from datetime import datetime
+from datetime import timedelta
+
+import os
+import time
+import pandas as pd
+from pprint import pprint
+
+try:
+    # Connect to PostgreSQL database
+    conn, engine = connect_to_db()
+
+    cur = conn.cursor(cursor_factory=psycopg2.extras.RealDictCursor)
+
+    # Get the list of all missions completed by turkers excluding onboarding and their first 500ft mission
+    # Filter out the missions that were already rewarded with a bonus in a previous run of this program. (These can be stored as an additional column in the mission_user_table)
```

Comment:
Can you bring these line lengths down to 100 characters (this applies to the whole file)